Core/HLE/sceKernelMemory.cpp

```diff
@@ -2301,3 +2301,96 @@ void Register_SysMemUserForUser()
 {
 	RegisterModule("SysMemUserForUser", ARRAY_SIZE(SysMemUserForUser), SysMemUserForUser);
 }
+
+
+struct HeapInformation : public KernelObject {
+	HeapInformation() {}
+	int uid;
+	int partitionId;
+	u32 size;
+	int flags;
+	u32 address;
+	std::string name;
+	BlockAllocator alloc;
+	static int GetStaticIDType() { return SCE_KERNEL_TMID_Fpl; }//  wrong
+	int GetIDType() const override { return SCE_KERNEL_TMID_Fpl; }//  wrong
```

Comment:
yeah, it won't make it. we can start adding some kernel stuff after 1.1 but for now I want to keep things the way they are to be safe.

1.1 will be soon though, I'm planning to release next weekend.